infrastructure/explorer/src/utils.js

```diff
@@ -61,38 +61,31 @@ export function getLocalAccountLink(address) {
 
 export function blockchainExplorerToken(token, account) {
     if (store.network === 'localhost') {
-        return `http://localhost:8000/${account}`;
+        return `${store.explorerLink}/${account}`;
     }
     const prefix = store.network === 'mainnet' ? '' : `${store.network}.`;
     const tokenAddress = window.syncProvider.tokenSet.resolveTokenAddress(token);
 
     if (tokenAddress != '0x0000000000000000000000000000000000000000') {
-        return `https://${prefix}etherscan.io/token/${tokenAddress}?a=${account}`;
+        return `${store.explorerLink}/token/${tokenAddress}?a=${account}`;
     } else {
-        return `https://${prefix}etherscan.io/address/${account}`;
+        return `${store.explorerLink}/address/${account}`;
     }
 }
 
 export function getBlockchainExplorerTx(network) {
-    if (network === 'localhost') {
-        return 'http://localhost:8000';
-    }
-    if (network === 'mainnet') {
-        return 'https://etherscan.io/tx';
+    if (store.explorerVersion === '0.1.0') {
+        return `${store.explorerLink}/transactions`;
     }
 
-    return `https://${network}.etherscan.io/tx`;
+    return `${store.explorerLink}/tx`;
 }
 
 export function getBlockchainExplorerAddress(network) {
-    if (network === 'localhost') {
-        return 'http://localhost:8000';
-    }
-    if (network === 'mainnet') {
-        return 'https://etherscan.io/address';
+    if (store.explorerVersion === '0.1.0') {
+        return `${store.explorerLink}/accounts`;
```

Comment:
We may need this only if the L1 explorer doesn't have the same path structure. Could we verify? If the path structure is the same, I'd remove it